### ISSUE TITLE
Context Menu not accessible for low vision users who rely on zooming functionality

### ISSUE THREAD

**Issue by randall-krauskopf** [zorkow/context-menu#34]
The context menu sub menu positioning does not meet the [WCAG 1.4.10 Reflow](https://www.w3.org/TR/WCAG21/#reflow) specification which is preventing consumers of this project from meeting accessibility standards.

## Demo of Issue
When dimensions are set at 320x256 you can see the menu overlap that hides information from the user.

<details><summary>Video</summary>
<p>

https://github.com/user-attachments/assets/441259e0-b98e-45f0-bc63-c02b96caa361

</p>
</details> 

## Example of non-overlap
This [sub-menu component](https://primer.style/react/storybook/?path=/story/components-actionmenu-features--submenus) stacks the new sub menus vertically, while the user now needs to scroll to view the new menus there is no loss of information.

<details><summary>Video</summary>
<p>

https://github.com/user-attachments/assets/311cf83c-2dcb-41cb-85ff-db0ce567a3ab

</p>
</details>
